tools/runtime_CLI.py

```diff
@@ -523,7 +533,12 @@ def parse_param_(field, bw):
             param = BmMatchParam(type = param_type,
                                  valid = BmMatchParamValid(key))
         elif param_type == BmMatchParamType.RANGE:
-            start, end = field.split("->")
+            try:
+                start, end = field.split("->")
```

Comment:
Can we use ..  or ... for the range? It is way more natural, in my view. We can add this in addition to ->, but I think that if we replace it, few people will notice. It is also easy to confuse -> with =>